components/nimbus/build.rs

```diff
@@ -26,5 +28,9 @@ pub fn main() {
         .status()
         .expect("Error generating Glean Rust bindings");
 
+    if !status.success() {
+        panic!("Failed generating Glean Rust bindings!")
```

Comment:
Is there a way to print some more information here in case a build fails for a developer? Something that can point them in some direction

Reply:
The glean_parser output would be the output that contains the information needed. Again, with my plan to switch to the glean-build crate rather than the script, the output should be available. If it isn't, I'll file a bug to see that the important info makes it into stdout.